test/test_proxyweb.py

```
@@ -0,0 +1,1511 @@
+        Attempts to load the table page (to refresh CSRF) and delete the row identified by `TEST_USER`. Any errors during cleanup are suppressed so teardown does not raise.
+        """
+        try:
+            self.s.get(f"/{self.SERVER}/{self.DB}/{self.TABLE}/")
+            self.s.post_json("/api/delete_row", {
+                "server":   self.SERVER,
+                "database": self.DB,
+                "table":    self.TABLE,
+                "pkValues": {"username": self.TEST_USER},
+            })
+        except Exception:
+            pass
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)